LondonBritishLibrary/orient/BLorient757.xml

```diff
@@ -56,59 +159,90 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">142</measure>
-                                    
-                                    
-                                    
-                                    
-                                    <dimensions type="outer" unit="mm">
-                                        <height>178</height>
-                                        <width>159</width>
+
+
+                                    <dimensions type="outer" unit="in">
+                                        <height>7</height>
+                                        <width>6.25</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+
+
                             </supportDesc>
                             <layoutDesc>
-                                <layout columns="2" writtenLines="20 22" cert="high">
-                           <note>except <locus from="134" to="142"/> where the number is larger.</note>
+                                <layout columns="2" writtenLines="20 22">
+                           <note>On <locus from="134" to="142"/> the number of
```

Comment:
I would maybe have a second layout with this locus range and the note.

Reply:
Probably a difference between the number of actual written lines and the number of "line areas" (space for one written line delimited by two ruled lines) should be considered, if possible. If the scribe squeezed more written lines into the same number of line areas (as the result, the size of the script had to be diminished), this should be only commented, in my opinion.